src/three-render-objects.js

```diff
@@ -298,7 +306,23 @@ export default Kapsule({
     }
 
     state.renderer.setSize(state.width, state.height);
-    state.camera.aspect = state.width/state.height;
+
+    if (state._cameraType === 'perspective') {
```

Comment:
Would it perhaps be cleaner to derive this from the camera object itself, eg:
```
state.camera.type === 'PerspectiveCamera' ? ...
```

You would also not need to maintain a spare `state._cameraType` attribute.

Reply:
ah, didn't know this was available, will use